index.html

```diff
@@ -1078,6 +1078,28 @@
   </div>
   <!-- ends Text watermark -->
 
+  <!-- Transparent Image Watermark -->
+  <span data-toggle="modal" data-target="#image_watermark"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Overlay image watermark">Overlay image watermark on video</button></span>
+  <div id="image_watermark" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Overlay image watermark on video</h3>
+          <p><code>ffmpeg -i <i>input_video file</i> -i <i>input_image_file</i> -filter_complex overlay=main_w-overlay_w-5:5 <i>output_file</i></code></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_video_file</i></dt><dd>path, name and extension of the input video file</dd>
+            <dt>-i <i>input_image_file</i></dt><dd>path, name and extension of the image file</dd>
+            <dt>-filter_complex overlay=main_w-overlay_w-5:5</dt><dd>This calls the overlay filter and sets x and y coordinates for the position of the watermark on the video. Instead of hardcoding specific x and y coordinates, <code>main_w-overlay_w-5:5</code> uses relative coordinates to place the watermark in the upper right hand corner, based on the width of your input files. Please see the <a href="https://www.ffmpeg.org/ffmpeg-all.html#toc-Examples-102">ffmpeg documentation for more examples.</a></dd>
+            <dt><i>output_file</i></dt><dd>path, name and extension of the output file.</dd>
```

Comment:
Like for the `-i input_video_file` and `-i input_image_file`, I suggest to not add a period at the end, as this is not a full sentence.